ui/src/containers/Header/index.tsx

```diff
@@ -82,6 +90,45 @@ const Header: React.FC = observer(() => {
           </TextContent>
         </Grid>
       </Modal>
+
+      <Modal
+        variant={ModalVariant.small}
+        title="Sign In:"
+        isOpen={isSignInModalOpen}
+        onClose={() => setIsSignInModalOpen(false)}
+        actions={[
+          <Grid key="close-button">
+            <GridItem offset={10}>
+              <Button
+                style={{ marginLeft: '0.7em' }}
+                key="cancel"
+                variant="secondary"
+                onClick={() => setIsSignInModalOpen(false)}
+              >
+                Close
+              </Button>
+            </GridItem>
+          </Grid>
+        ]}
+      >
+        <Grid>
+          {providers.values.slice(0, 1).map((provider: IProvider) => (
+            <GridItem key={provider.name} offset={3} span={6} className="header-sigin-button">
+              <Button
```

Comment:
I just had a question, if the button is clicked once is the button disabled ?
Also what are we doing if there is some error coming from the auth server, are we setting any alerts ??

Reply:
it does't disable ,there is no issue if it is clicked more than once